src/components/basket/Basket.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import BasketProduct from './BasketProduct';
+import BasketPriceDisplay from './BasketPriceDisplay';
+import BasketEmptyMessage from './BasketEmptyMessage';
+import SubNavigation from '../navs/SubNavigation';
+import BasketCheckoutButton from './BasketCheckoutButton';
+import BasketOrderSent from './BasketOrderSent';
+
+function Basket(props) {
+    const totalAmount = props.orderAmount + props.deliveryPrice;
+    const deliveryPrice = props.deliveryPrice;
+
+    function ordersHandler(dishId, quantity, price, action) {
+        props.receiver(dishId, quantity, price, action);
+    }
+
+    function oldOrdersHandler() {
+        props.oldOrders();
+    }
+
+    function checkoutHandler(order) {
+        const date = new Date();
+        const minutes = date.getMinutes() < 10 ? '0' + date.getMinutes() : '' + date.getMinutes();
```

Comment:
Moving date formating to own module would allow you to re-use the code here and avoid duplication